modules/sso-rm-openjdk/configure.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+set -u
+set -e
+
+if ! rpm -q java-1.8.0-openj9-devel; then
+  exit
+fi
+
+for pkg in java-1.8.0-openjdk-devel \
+       java-1.8.0-openjdk-headless \
+       java-1.8.0-openjdk; do
+    if rpm -q "$pkg"; then
+        rpm -e --nodeps "$pkg"
```

Comment:
Can't the `--nodeps` option break some dependency, that would otherwise work? Can't the intended removal be performed on yum level? (`yum -y remove java-1.8.0-openjdk{,-devel,-headless}`)

Reply:
This is the only option we currently have and has been copied from https://github.com/jboss-openshift/cct_module/blob/master/jboss/container/java/rm-openjdk/configure.sh without changes

Reply:
Fair enough. Not nice, but so be it (I will check with @jmtd why the `rpm -e --nodeps` construct is required there).